Algorithm/Number_theory/readme.md

```diff
@@ -48,5 +48,5 @@
 - Modular Exponentiation ----> [C++](/Code/C++/modular_exponentiation.cpp)
 - Palindrome Number ---> [C++](/Code/C++/palindrome_number.cpp) | [Python](/Code/Python/palindrome_number.py)
 - Taylor's Series of e^x ----> [Java](/Code/Java/Taylorseries.java)
-- To find all prime number from 2 to N ----> [C++](/Code/C++/Sieves_prime.cpp)
-
+- To find all prime number from 2 to N ----> [C++](Code/C++/Sieves_prime.cpp)
```

Comment:
add / before Code in  To find all prime number from 2 to N ----> [C++](Code/C++/Sieves_prime.cpp)